app/services/authorization/user_permissible_service.rb

```diff
@@ -48,15 +49,17 @@ def allowed_in_entity?(permission, entities_to_check, entity_class)
 
     def allowed_in_any_entity?(permission, entity_class, in_project: nil)
       perms = contextual_permissions(permission, context_name(entity_class))
+      return false unless authorizable_user?
       return true if admin_and_all_granted_to_admin?(perms)
 
       # entity_class.allowed_to will also check whether the user has the permission via a membership in the project.
+      # TODO: ^--- thats' actually not the case weirdly...
       allowed_scope = entity_class.allowed_to(user, perms)
 
       if in_project
-        allowed_scope.exists?(project: in_project)
+        allowed_in_single_project?(perms, in_project) || allowed_scope.exists?(project: in_project)
```

Comment:
I have to figure out why, but the `WorkPackage.allowed_to` scope does not properly check project permissions. This is a band-aid fix to get the method working and I will fix the scope afterwards and remove this